### PR TITLE
narrow down CORS config

### LINKED ISSUE #40: Narrow down CORS configuration

### PR DESCRIPTION
**Pull request by niklaslong**
closes #40

### CODE REVIEW

**Comment by niklaslong** [comment]
@marcoow is this the correct origin?

File: apps/breethe_web/config/config.exs [hunk 1]
```diff
@@ -35,6 +35,8 @@ config :mime, :types, %{
 config :ja_serializer, key_format: {:custom, JsonApiKeys, :camelize, :underscore}
 config :ja_serializer, type_format: {:custom, JsonApiKeys, :camelize}
 
+config :cors_plug, origin: ["http://breethe.app"]
```
Comment (marcoow) [inline apps/breethe_web/config/config.exs]: https

Comment (marcoow) [reply apps/breethe_web/config/config.exs]: We should also allow `http://dev.breethe.app` so we can use that for developing locally

Comment (niklaslong) [reply apps/breethe_web/config/config.exs]: fixed

**Pull request merged**